blazor/maps/getting-started.md

```diff
@@ -17,42 +17,41 @@ This section briefly explains about how to include [Blazor Maps](https://www.syn
 
 ## Create a new Blazor App in Visual Studio
 
-You can create **Blazor Server App** or **Blazor WebAssembly App** using Visual Studio in one of the following ways,
+You can create a **Blazor Server App** or **Blazor WebAssembly App** using Visual Studio via [Microsoft Templates](https://learn.microsoft.com/en-us/aspnet/core/blazor/tooling?view=aspnetcore-7.0) or the [Syncfusion Blazor Extension](https://blazor.syncfusion.com/documentation/visual-studio-integration/template-studio).
 
-* [Create a Project using Microsoft Templates](https://learn.microsoft.com/en-us/aspnet/core/blazor/tooling?view=aspnetcore-7.0&pivots=windows)
+## Install Syncfusion Blazor Maps and Theme NuGet in the App
```

Comment:
Did checked this component without Theme NuGet?

Reply:
Yes, checked this component without Theme NuGet also Theme NuGet was not needed . So Theme NuGet was removed from this PR.